ex23_ch4.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: get input from user and turn input to lowercase
+turn_key = input("Is the car silent when you turn the key? Enter \"yes\" or \"no\": ")
+normalized_turn_key = turn_key.lower()
+
+#step2: write first conditional based on answer from input
+if turn_key == "yes":
+    battery = input("Are the battery terminals corroded? Enter \"yes\" or \"no\": ")
+    normalized_battery = battery.lower()
+    if battery == "yes":
+        print("Clean the terminals and try starting again.")
+    else: 
+        print("Replace the cables and try it again.")
+
+#step3: write second conditional based on input from user
+elif turn_key == "no":
+    click_sound = input("Does the car make a clicking noise? Enter \"yes\" or \"no\": ")
+    normalized_click_sound = click_sound.lower()
+    if click_sound == "yes":
+        print("Replace the battery.")
+    elif click_sound == "no":
```

Comment:
I intentionally use "elif" here. Is this the best option? I wouldn't use "else" here, correct?

Reply:
I agree here. The only thing to note is what happens if they don't enter `yes` or `no`. However, I think it's fine for the purposes of this exercise.